plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -197,27 +204,31 @@ func (adx *AzureDataExplorer) createAzureDataExplorerTable(ctx context.Context,
 	if _, err := adx.client.Mgmt(ctx, adx.Database, createTableMappingstmt); err != nil {
 		return err
 	}
-
 	return nil
 }
 
 func (adx *AzureDataExplorer) Init() error {
 	if adx.Endpoint == "" {
-		return errors.New("Endpoint configuration cannot be empty")
+		return errors.New("endpoint configuration cannot be empty")
 	}
 	if adx.Database == "" {
-		return errors.New("Database configuration cannot be empty")
+		return errors.New("database configuration cannot be empty")
 	}
 
 	adx.MetricsGrouping = strings.ToLower(adx.MetricsGrouping)
 	if adx.MetricsGrouping == singleTable && adx.TableName == "" {
-		return errors.New("Table name cannot be empty for SingleTable metrics grouping type")
+		return errors.New("table name cannot be empty for SingleTable metrics grouping type")
 	}
 	if adx.MetricsGrouping == "" {
 		adx.MetricsGrouping = tablePerMetric
 	}
+
+	if adx.IngestionType == "" {
+		adx.IngestionType = queuedIngestion
+	}
```

Comment:
It would be nice to check the type here. E.g.
```suggestion
	if adx.IngestionType == "" {
		adx.IngestionType = queuedIngestion
	}
	if !choice.Contains(adx.IngestionType, []string{"queued", "managed"}) {
		return fmt.Errorf("unknown ingestion type %q", adx.IngestionType)
	}
```